salt/matchers/grain_pcre_match.py

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8 -*-
+'''
+This is the default grains PCRE matcher.
+
+NOTE: These functions are converted to methods on the Matcher class during master and minion startup.
+This is why they all take `self` but are not defined inside a `class:` declaration.
+'''
+from __future__ import absolute_import, print_function, unicode_literals
+
+import logging
+from salt.defaults import DEFAULT_TARGET_DELIM  # pylint: disable=3rd-party-module-not-gated
+
+import salt.utils.data  # pylint: disable=3rd-party-module-not-gated
+
+log = logging.getLogger(__name__)
+
+
+def match(self, tgt, delimiter=DEFAULT_TARGET_DELIM):
+    '''
+    Matches a grain based on regex
+    '''
+    log.debug('grains pcre target: %s', tgt)
```

Comment:
Maybe we should have here a bit more verbose message that mentions matcher explicitly. When lots of debug/trace is happening, then it is easier to grasp this kind of data.